build_library/grub_install.sh

```diff
@@ -71,6 +73,9 @@ cleanup() {
     if [[ -b "${LOOP_DEV}" ]]; then
         sudo losetup --detach "${LOOP_DEV}"
     fi
+    if [[ -e "${BUILD_LIBRARY_DIR}/grub.cfg" ]]; then
```

Comment:
oh, I see.... even if you stick with two files you need to use a temp directory, you should absolutely not be modifying files in git.